sim/firesim-lib/src/main/cc/fesvr/firesim_tsi.h

```diff
@@ -48,5 +52,7 @@ class firesim_tsi_t final : public testchip_tsi_t {
 private:
   size_t idle_counts;
   bool is_busy;
+  bool is_loaded;
```

Comment:
Can you add some comments describing what these new fields indicate?

Reply:
I think perhaps a better naming is `is_loaded_in_dut`/`is_loaded_in_fesvr`? Not sure

Reply:
Good catch, yes the names should change. I think to something like `fesvr_pre_reset_xacts_sent` and `target_synced_all_xacts`.

Reply:
Maybe `is_loaded_in_host` and `is_loaded_in_target`?